setup32.cfg

```diff
@@ -0,0 +1,18 @@
+[aliases]
+# python2.7 has upgraded unittest and it is no longer compatible with some
+# of our tests, so we run all through nose
+test = nosetests
+
+[nosetests]
+# nosetests skips test files with the executable bit by default
+# which can silently hide failing tests.
+# There are no executable scripts within the scikit-learn project
+# so let's turn the --exe flag on to avoid skipping tests by
+# mistake.
+exe = 1
+cover-html = 1
+cover-html-dir = coverage
+cover-package = sklearn
+
+detailed-errors = 1
+with-doctest = 0
```

Comment:
I suspect that the reason that we need the setup32.cfg file is really because of that last line, that turns off doctests.

I find it quite horrible to have to create a whole cfg file just for this. The reason is that our standard cfg file forces nsoe to run doctests (which is in general good).

I guess that you could try to simply have nose ignore this file, by setting the environment variable: NOSE_IGNORE_CONFIG_FILES (see http://nose.readthedocs.org/en/latest/usage.html)